homeassistant/components/media_player/plex.py

```diff
@@ -79,22 +161,28 @@ def setup_platform(hass, config, add_devices_callback, discovery_info=None):
     else:
         return
 
-    setup_plexserver(host, token, hass, add_devices_callback)
+    setup_plexserver(host, token, hass, optional_config, add_devices_callback)
+
 
+def set_group_members(hass, group_entity_id, member_entity_id_list):
```

Comment:
This is not allowed. This is not part of how groups work in Home Assistant. Please remove.